modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/AssignmentsTracker.java

```diff
@@ -97,7 +97,7 @@ public void startTrack() {
 
                     TablePartitionId grpId = TablePartitionId.fromString(strKey);
 
-                    Set<Assignment> assignments = ByteUtils.fromBytes(entry.value());
+                    Set<Assignment> assignments = Assignments.fromBytes(entry.value()).nodes();
 
                     groupAssignments.put(grpId, assignments);
```

Comment:
Why do we still have Set<Assignment> instead of Assignments in groupAssignments?

Reply:
I'm planning to add meta-information to the Assignments class that won't be required for tracking. So this class is only for storage and communication

Reply:
Make sense.